Source/EventFlow/EventStores/Files/FilesEventPersistence.cs

```diff
@@ -40,6 +40,7 @@ public class FilesEventPersistence : IEventPersistence
         private readonly IFilesEventLocator _filesEventLocator;
         private readonly AsyncLock _asyncLock = new AsyncLock();
         private readonly string _logFilePath;
+        private static AsyncLock _logFileSemaphore = new AsyncLock();
```

Comment:
EventFlow only uses static field for stuff that could be shared between different domains as EventFlow is design to be used as multiple instances within the same process.